README.md

```diff
@@ -49,6 +69,117 @@ ECx.load('jdkknkkbebbapilgoeccciglkfbmbnfm');
 
 ```
 
+## ECx APIs
+
+- `ECx.load(extensionId: IExtension['id']): Promise<IExtension>`
+- `ECx.unload(extensionId: IExtension['id']): void`
+- `ECx.setConfiguration(configuration: Configuration = {}): Promise<ECx>`
+- `ECx.isLoaded(extensionId: IExtension['id']): boolean`
+- `ECx.isUpToDate(extensionId: IExtension['id']): Promise<boolean>`
+- `ECx.get(extensionId: IExtension['id']): Promise<IExtension>`
+- `ECx.sendEvent(event: ExtensionEventMessage): void`
+
+## Add extension API binding
+
+Add missing chrome APIs is seamless.
+
+In the folder `src/common/apis/`, add `<namespace>.ts` file is missing.
+It should include two enums: `Methods` and `Events` like this:
+
+```ts
+// cookies.ts
+
+export enum Methods {
+  Get = 'get',
+  GetAll = 'get-all',
+  Set = 'set',
+  Remove = 'remove',
+  GetAllCookieStores = 'get-all-cookie-stores',
+}
+
+export enum Events {
+  OnChanged = 'onChanged',
+}
+```
+
+The register it in `src/common/apis/index` and `enum Api` in `src/common/index`
+
+In the folder `src/renderer/api/`, add `<namespace>.ts` file is missing.
+It should be like this:
+
+```ts
```

Comment:
So this is intended to contributors of this lib, not to users, right? Might be better in a `Contributing` section.

Reply:
You can directly link to the file implemented :)

Reply:
👍